rapids-cmake/cpm/rmm.cmake

```
@@ -73,13 +73,28 @@ function(rapids_cpm_rmm)
   include("${rapids-cmake-dir}/cpm/detail/generate_patch_command.cmake")
   rapids_cpm_generate_patch_command(rmm ${version} patch_command build_patch_only)
 
+  # Use `SOURCE_SUBDIR cpp` if RMM version is 25.06 or higher
+  include("${rapids-cmake-dir}/cmake/parse_version.cmake")
+  rapids_cmake_parse_version(MAJOR ${version} major_version)
+  rapids_cmake_parse_version(MINOR ${version} minor_version)
+  if(major_version GREATER 25 OR (major_version EQUAL 25 AND minor_version GREATER_EQUAL 6))
```

Any reason not to do this?

```suggestion
  if(version VERSION_GREATER_EQUAL 25.06)
```

---

Good idea. I just need to write like it's CMake and not like it's C preprocessor.